src/lib/jsonParse/jsonUpdateContextAttributeRequest.cpp

```diff
@@ -63,6 +63,9 @@ static std::string attributeValue(const std::string& path, const std::string& va
   //       save the 'typeFromXmlAttribute', a ContextAttribute has been added to 
   //       UpdateContextAttributeData.
   //
+  // FIXME P4 #1862: typeFromXmlAttribute was removed. The comment abov is obsolete and maybe
```

Comment:
@kzangeli I'd need the new version of the above comment (or maybe it can be removed)?